proto/pulumi/resource.proto

```diff
@@ -281,6 +297,18 @@ message TransformInvokeOptions {
     map<string, bytes> plugin_checksums = 4;
 }
 
+// ResourceHookRequest is the request object for resource hook callbacks in CallbackInvokeRequest.
+message ResourceHookRequest {
+    string urn = 1;
+    string id = 2;
+    google.protobuf.Struct outputs = 3;
```

Comment:
Just outputs? Why not inputs as well, the before_create will only really have inputs filled in.

Reply:
Yeh a bit confusing, but the same as for analyzers.